.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib"
+}
```

Comment:
This is vscode ts version setting.

Since vscode comes with internal ts, different version of vscode can cause syntactic error.